app/serializers/restricted/user_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class Restricted::UserSerializer < ActiveModel::Serializer
+  embed :ids, include: true
+  attributes :id, :restricted, :username, :email, :email_when_proposal_closing_soon, :email_missed_yesterday,
```

Comment:
I ended up not being able to care about extending modules like this to extract it out, as it ended up being more of a pain than it was worth.